README.md

```diff
@@ -21,7 +22,8 @@
 ## Live Demo
```

Comment:
- [ ] Kindly check your deployment link. It seems the link is for an older version of your app.

Reply:
hello Mr @phelian23 i have fixed all the required error can i have a reviews again please